File: app/src/am_huynn/com/example/asian/MainActivity.java
package com.example.asian;

import android.content.Intent;
import android.os.Bundle;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

import com.example.asian.ui.ExerciseFragmentActivity;
import com.example.asian.ui.FacebookActivity;
import com.example.asian.ui.InformationActivity;
import com.example.asian.ui.MathActivity;

public class MainActivity extends AppCompatActivity {
    private Button mBtnToFacebook;
    private Button mBtnToMath;
    private Button mBtnToInformation;
    private Button mBtnExerciseFragment;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        initView();
        initListener();
    }

    private void initView() {
        mBtnToFacebook = findViewById(R.id.btnFacebook);
        mBtnToMath = findViewById(R.id.btnMath);
        mBtnToInformation = findViewById(R.id.btnInformation);
        mBtnExerciseFragment = findViewById(R.id.btnExerciseFragment);
    }

    private void initListener() {
        mBtnToFacebook.setOnClickListener(view -> {
            Intent intent = new Intent(this, FacebookActivity.class);
            startActivity(intent);
        });

        mBtnToMath.setOnClickListener(view -> {
            Intent intent = new Intent(this, MathActivity.class);
            startActivity(intent);
        });

        mBtnToInformation.setOnClickListener(view -> {
            Intent intent = new Intent(this, InformationActivity.class);
            startActivity(intent);
        });

        mBtnExerciseFragment.setOnClickListener(view -> {
            Intent intent = new Intent(this, ExerciseFragmentActivity.class);
            startActivity(intent);
        });
    }
}

File: app/src/am_sonhk/com/example/asian/views/MainActivity.kt
package com.example.asian.views

import android.content.Intent
import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity
import com.example.asian.DatabaseEx
import com.example.asian.databinding.MainActivityBinding

class MainActivity : AppCompatActivity() {
    lateinit var binding: MainActivityBinding

    private fun initListioner() {
        with(binding) {
            btnDatabase.setOnClickListener {
                startActivity(Intent(this@MainActivity, DatabaseEx::class.java))
            }
        }

    }

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = MainActivityBinding.inflate(layoutInflater)
        setContentView(binding.root)
        initListioner()
    }
}